models/pyproject.toml

```diff
@@ -46,7 +46,7 @@ dependencies = [
   "anemoi-utils>=0.1.9",
   "einops>=0.6.1",
   "hydra-core>=1.3",
-  "torch>=2.2",
```

Comment:
Is this increment based on testing?
Given that training requires models, would it make sense to update in the training toml?

Reply:
This increment is based on testing.  With torch 2.2, `AttributeError: module 'torch.nn' has no attribute 'attention'` is raised in multiple tests.

I'm not sure about the torch version required by `models`. But since `models` requires `training`, it inherits the `models` dependencies. I would leave it up to the software engineers to decide, whether `training` needs to declare its dependency on `torch` (and others) in this case at all.

Reply:
Let's leave it as this for now.